Jenkinsfile

```diff
@@ -1011,6 +1011,9 @@ def installTools() {
     retryWithSleep(retries: i, seconds: 5, backoff: true){ sh(label: "Install Terraform ${TERRAFORM_VERSION}", script: ".ci/scripts/install-terraform.sh") }
     retryWithSleep(retries: i, seconds: 5, backoff: true){ sh(label: "Install Mage", script: "make mage") }
   } else {
+    // Install python3 with the specific step, even though install-tools.bat will verify if it's there anyway.
+    // TODO: as soon as python3 is installed in the CI Workers we will be able to remove the line below.
+    installTools([ [tool: 'python3', version: '3.8', exclude: 'rc'] ])
```

Comment:
Should this be a retry?

Reply:
That's another PR that I'll need to backport once it's merged :/